lib/wordpress_ex/users.ex

```diff
@@ -0,0 +1,40 @@
+defmodule WordpressEx.Users do
+  @moduledoc """
+  Wordpress User endpoints
+  """
+  import WordpressEx.Client, only: [get: 2]
+  import WordpressEx.StructComposer, only: [compose: 2]
+  alias WordpressEx.Model.User
+
+  @doc """
+  Get a list of categories
```

Comment:
```suggestion
  Get a list of users
```

I think this was just a copy/paste leftover. Can we change this to `users`?

Reply:
Yep, my bad! Thanks for the catch